### ISSUE TITLE
creating first story update doesn't update the UI

### ISSUE THREAD

**Issue by madhums** [compassionprojects/socialchange#61]
When creating the first story update of a story, doesn't update the UI. It's seen only after refreshing a page.

Steps to reproduce:

1. Go to a story that doesn't have a story update
2. Click on "new story update" and create a story update
3. I expect to see the story update but nothing happens
4. Refresh the page, then it appears.
